// TryCatch statements
try this
    throw {"msg": "Oh no", "code": 400};
catch as e do
    $print(e);
end

try this
    a = [1, 2, 3];
    a[-4] = -1;
catch as e do
    $print(e);
end

fun foo(a, b)
    try this
        return a * b;
    catch as e do
        $print(e);
    end
end

$foo(2, 4);
